### PR TITLE
Ad story 37

### PR DESCRIPTION
**Pull request by adumortier**
- added the routes for the `Admin::Merchants` `index` and `show`
- added an `Admin::Merchants` controller 
- added an `Admin::Merchants` `index` and `show` view pages
- added a test for the `Admin::Merchants` `show` page

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -42,9 +42,11 @@
...
   end
 
-  namespace :admin do
-    get '/', to: 'dashboard#index'
-  end
+    namespace :admin do
+      get '/', to: 'dashboard#index'
+      get '/merchants', to: 'merchants#index'
+      get '/merchants/:id', to: 'merchants#show'
+    end
```
Comment (TravisBorgsmiller) [inline config/routes.rb:50]: I really like these routes

File: app/controllers/admin/merchants_controller.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+  def index 
+     @merchants = Merchant.all
+  end
+
+  def show
+    @merchant = Merchant.find(params[:id])
+  end
+
+end
```
Comment (TravisBorgsmiller) [inline app/controllers/admin/merchants_controller.rb:11]: Love this admin merchants_controller!

**Review by TravisBorgsmiller — APPROVED** [review]

File: config/routes.rb (hunk 1 above)
Comment (KellyIB) [reply config/routes.rb:50]: Agree with Travis

**Review by KellyIB — APPROVED** [review]
Like the merchants controller inside the admin folder.

**Pull request merged**